styleguide/webpack.config.js

```diff
@@ -15,15 +15,20 @@ module.exports = {
       {
         test: /\.(jpeg|ico)$/,
         exclude: /node_modules/,
-        loader: 'url-loader?limit=10000',
+        use: [{
+          loader: 'url-loader',
+          options: {
+            limit: 10000,
+          },
+        }],
       },
       {
         test: /\.svg$/,
         include: [
           /src\/svgs\//,
         ],
         use: [
-          { loader: 'svg-sprite-loader' },
+          { loader: 'svg-sprite-loader', options: {} },
```

Comment:
Is this addition necessary?

Reply:
Yeah, `svg-sprite-loader` has a bug with Webpack 5 -- this fix should help any future endeavor at upgrading Webpack. It was identified a while ago and no one has fixed it.

I'm actually now re-considering SVG sprite loading. We don't actually get any real benefit from an SVG sprite sheet in new React code. We might even be able to get better benefits if we take SVG files and dynamically generate JSX for those or something.